app.py

```diff
@@ -1,32 +1,43 @@
-from flask import Flask, request, jsonify
+from flask import Flask, request, jsonify, json
+from pymongo import MongoClient
 
 app = Flask(__name__)
 
+client = MongoClient('mongodb://localhost:27017/')
+db = client['to-do-list']
+collection = db['to-do-list']
+
 items = []
 
 
 @app.route('/', methods=['GET'])
 def hello_world():
     return 'Welcome, this is my todo list!'
 
+
 @app.route('/todo', methods=['POST'])
 def create():
-    items.append(request.json['data'])
-    return jsonify('items', items)
+    items.append(request.json['data']) #will be deleted later
+    req_data = request.get_json()
+    collection.insert_one(req_data).inserted_id
+    return "204"
 
 @app.route('/todo', methods=['GET'])
 def getall():
     return jsonify('items', items)
```

Comment:
I presume if you are able to insert a data to mongo (which supposedly success). Then, modify `get` would be easy.